wpimath/src/main/native/cpp/geometry/Translation3d.cpp

```diff
@@ -19,12 +19,21 @@ Translation3d::Translation3d(units::meter_t distance, const Rotation3d& angle) {
   m_z = rectangular.Z();
 }
 
+Translation3d::Translation3d(const Eigen::Vector3d& vector)
+    : m_x(units::meter_t(vector.x())),
+      m_y(units::meter_t(vector.y())),
+      m_z(units::meter_t(vector.z())) {}
```

Comment:
```suggestion
    : m_x{units::meter_t{vector.x()}},
      m_y{units::meter_t{vector.y()}},
      m_z{units::meter_t{vector.z()}} {}
```

Reply:
what's the reason for methods like
```cpp
constexpr Translation2d::Translation2d(units::meter_t x, units::meter_t y)
    : m_x(x), m_y(y) {}
```
using direct initialization instead of list initialization?

Reply:
It's legacy code from 2020.

Reply:
guessing i should change that too

Reply:
Yea, if you would please.